src/extensions/extensionsHelper.ts

```diff
@@ -177,15 +177,15 @@ export function populateDefaultParams(
   const newParams = paramVars;
 
   for (const param of paramSpecs) {
-    if (!paramVars[param.param]) {
-      if (param.default !== undefined && param.required) {
-        newParams[param.param] = param.default;
-      } else if (param.required) {
-        throw new FirebaseError(
-          `${param.param} has not been set in the given params file` +
-            " and there is no default available. Please set this variable before installing again."
-        );
-      }
+    if (paramVars[param.param]) continue;
```

Comment:
Changes look good. The first line could be an unnecessary change so can probably leave as it was.